### PR TITLE
finished nav bar with some tabs

### PR DESCRIPTION
**Pull request by tiffanydoh**

### CODE REVIEW

File: app/javascript/components/pages/About.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+export class About extends Component {
+  render() {
+    return (
+      <div>About</div>
+    )
+  }
+}
+
+export default About
```
Comment (VonTeacher) [inline app/javascript/components/pages/About.js:11]: Let's add an EOF empty line.

File: app/javascript/components/pages/SongEdit.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+export class SongEdit extends Component {
+  render() {
+    return (
+      <div>SongEdit</div>
+    )
+  }
+}
+
+export default SongEdit
```
Comment (VonTeacher) [inline app/javascript/components/pages/SongEdit.js:11]: Let's add an EOF empty line.

File: app/javascript/components/pages/SongIndex.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+export class SongIndex extends Component {
+  render() {
+    return (
+      <div>SongIndex</div>
+    )
+  }
+}
+
+export default SongIndex
```
Comment (VonTeacher) [inline app/javascript/components/pages/SongIndex.js:11]: Let's add an EOF empty line.

File: app/javascript/components/pages/SongNew.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+export class SongNew extends Component {
+  render() {
+    return (
+      <div>SongNew</div>
+    )
+  }
+}
+
+export default SongNew
```
Comment (VonTeacher) [inline app/javascript/components/pages/SongNew.js:11]: Let's add an EOF empty line.

File: app/javascript/components/pages/SongShow.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+export class SongShow extends Component {
+  render() {
+    return (
+      <div>SongShow</div>
+    )
+  }
+}
+
+export default SongShow
```
Comment (VonTeacher) [inline app/javascript/components/pages/SongShow.js:11]: Let's add an EOF empty line.

**Review by WalkerTexas-coder — DISMISSED** [review]
This is really good. Router can be the top level tag that wraps around header and everything else without <> </> tags. That can be taken care of in the next PR.

File: app/javascript/components/components/Header.js [hunk 6]
```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react'
-import { Nav, NavItem } from 'reactstrap'
+import { Collapse, DropdownItem, DropdownMenu, DropdownToggle, Nav, Navbar, NavbarBrand, NavbarText, NavbarToggler, NavItem, NavLink, UncontrolledDropdown } from 'reactstrap'
```
Comment (VonTeacher) [inline app/javascript/components/components/Header.js:2]: For developer legibility, let's break this into a multi-line object, like:
```js
import {
  Collapse,
  DropdownItem,
  DropdownMenu,
  ...etc
} from 'reactstrap'
```

**Review by VonTeacher — CHANGES_REQUESTED** [review]
Some basic changes requested. Let's touch on some things I saw in our mentor session today. 👍

**Comment by VonTeacher** [comment]
Additionally, here's what I'm seeing when I'm logged out:

out
versus logged in:

in

Where did my options go? 😢

**Review by SunkissedQueen — APPROVED** [review]
Looks good

**Review by VonTeacher — APPROVED** [review]
Looks great! 👍

**Pull request merged**